src/ssl.c

```diff
@@ -18660,7 +18664,7 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
     {
         int ret;
 
-        WOLFSSL_ENTER("wolfSSL_MD5_Update");
+        WOLFSSL_ENTER("MD5_Update");
```

Comment:
Don't make this change.
(Leave wolfSSL prefix on.)
Other hash functions should have wolfSSL prefix though.

Reply:
What's the reasoning for this? By changing it the name matches the other MD5 and other hashing compatibility functions.

Reply:
I don't know why this PR was started but I assume it was, in part, to make the WOLFSSL_ENTER parameter text consistent with the name of the function.
Changing wolfSSL_MD5_Update to MD5_Update is a step backwards. The other WOLFSSL_ENTER calls for other hash functions should be fixed too.

If you don't want to take this work on in this PR, I'll resolve this comment and it can happen in another PR down the line sometime.

Reply:
It started because of some typos between the function name and the entry logging function. Then I started cleaning up the mis-matches.